llvm/lib/Transforms/IPO/Attributor.cpp

```diff
@@ -2963,6 +2963,132 @@ bool Attributor::shouldSeedAttribute(AbstractAttribute &AA) {
   return Result;
 }
 
+// For now: argument can be put in the struct if it's write only and
+// has no aliases.
+static bool canBeComapctedInAStruct(const Argument &Arg, Attributor &A,
```

Comment:
Typo 'Comapcted'. But I don't think this is the right phrasing. Something more like canConvertOutArgToReturn?